contracts/Response.sol

```diff
@@ -189,16 +228,18 @@ contract Response {
     qs.highestLowestSupporterIdx = sIdx;
     setQuestionState(qIdx, qs);
   }
-
+  // QUESTION: why does parameter address exist/not have a name?
   function receiveApproval(address from, uint256 value, address, bytes extraData) public {
+    // QUESTION: the sender of this transaction will be the AE token contract?
```

Comment:
The user will send a transaction that calls `approveAndCall` of our token contract and then the token contract will make an internal call to response app contract. So, `msg.sender` should be equal to address of our token contract. According to [this](https://github.com/aeternity/contraect/blob/9a9b41db268eb110f741f11914db316eb08ca9fa/contracts/HumanStandardToken.sol#L62), aeternity token contract always sends own address as the third argument, it shouldn't be necessary to check it.